validation/src/test/scala/hmda/validation/rules/ts/validity/V120Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.ts.TsEditCheckSpec
+import hmda.validation.rules.ts.validity.V120
+import org.scalacheck.Gen
+
+class V120Spec extends TsEditCheckSpec {
+
+  property("passes with properly formed number") {
+    forAll(tsGen) { ts =>
+      whenever(ts.contact.phone.length == 12) {
+        ts.mustPass
+      }
+    }
+  }
+
+  property("fail with other seperators") {
+    forAll(tsGen, badPhoneNumberGen) { (ts: TransmittalSheet, x: String) =>
+      val badContact = ts.contact.copy(phone = x)
+      val badTs = ts.copy(contact = badContact)
+      badTs.mustFail
+    }
+  }
+
+  override def check: EditCheck[TransmittalSheet] = V120
+
+  implicit def badPhoneNumberGen: Gen[String] = {
```

Comment:
Since this is used in 2 different tests, would it make sense to pull it out into something similar to the `BadValueUtils` trait that we have in the LAR tests?